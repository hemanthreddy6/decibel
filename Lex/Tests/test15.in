audio sound_3 = audio(PAN,0.8,440);
audio sound_4 = audio(SQUARE,0.5,220);

sound_3 = sound_3 * sound_4;
filtered_audio = sound_3[10:20];
play filtered_audio^3;
