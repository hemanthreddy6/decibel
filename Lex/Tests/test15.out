AUDIO
IDENTIFIER
=
AUDIO
(
PAN
,
FLOAT_LITERAL
,
INT_LITERAL
)
;
AUDIO
IDENTIFIER
=
AUDIO
(
SQUARE
,
FLOAT_LITERAL
,
INT_LITERAL
)
;
IDENTIFIER
=
IDENTIFIER
*
IDENTIFIER
;
IDENTIFIER
=
IDENTIFIER
[
INT_LITERAL
:
INT_LITERAL
]
;
PLAY
IDENTIFIER
^
INT_LITERAL
;
